lib/package.gd

```diff
@@ -691,6 +691,31 @@ DeclareGlobalFunction( "LoadPackageDocumentation" );
 ##
 ##  <Description>
 ##  loads the &GAP; package with name <A>name</A>.
+## <P/>
+##  As an example, the following loads the &GAP; package
+## <Package>Sonata</Package> (case insensitive) which provides methods for the
```

Comment:
Please correct this to `<Package>SONATA</Package>` to match the style chosen by authors. This will also trigger new Travis build which now includes making GAP manuals.